### PR TITLE
Integrate monitoring into piman.py

### PR DESCRIPTION
**Pull request by EveretteLi**

### CODE REVIEW

File: make_zipapp.sh [hunk 1]
```diff
@@ -8,14 +8,16 @@ mkdir build/piman.app
...
+cp -r hosts.csv monitoring/monitoring.config build
 mkdir build/piman.app/install
 mkdir build/logs
 cp piman.yaml build
 #git clone --depth 1 --single-branch https://github.com/raspberrypi/firmware.git firmware
 if [ ! -f firmware/boot/bootcode.bin ]
 then
-    git submodule init firmware
-    git submodule update --depth 1 firmware
+   git submodule init firmware
```
Comment (breed) [inline make_zipapp.sh]: oops it looks like you deleted some spaces by accident.

File: piman.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-private_number: 1
-server_address: 172.17.2.1
+private_number: 3
```
Comment (breed) [inline piman.yaml]: this file is not related to the change

File: piman.py [hunk 3]
```diff
@@ -106,6 +109,11 @@ def server():
     # when it is run, so after it is closed by a keyboard interrupt, the above
     # lines run closing the rest of the threads. ntp_thread will already be stopped
 
+# start monitoring thread
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
+    monitor_thread.start()
```
Comment (breed) [inline piman.py]: if you are just going to do a start() then join(), you don't need a thread. you can just call it.

File: monitoring/monitoring_client.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
+"""
```
Comment (breed) [inline monitoring/monitoring_client.py]: why is this getting added, twice?

**Review by breed — APPROVED** [review]
you need to updated the install client script and probably at least one of the doc files.

**Review by breed — CHANGES_REQUESTED** [review]
sorry, i clicked the wrong button. please make the requested changes.

File: make_zipapp.sh (hunk 1 above)
Comment (EveretteLi) [reply make_zipapp.sh]: For the "install client script", do you mean by make_zipapp.sh or the .sh referenced by monitoring.service?
